lib/services/services.c

```diff
@@ -222,13 +222,23 @@ services__create_resource_action(const char *name, const char *standard,
     }
 
     if (strcasecmp(op->standard, PCMK_RESOURCE_CLASS_OCF) == 0) {
-        char *dirs = strdup(OCF_RA_PATH);
+        char *dirs = NULL;
         char *dir = NULL;
         char *buf = NULL;
         struct stat st;
 
-        if (pcmk__str_empty(dirs)) {
-            free(dirs);
+        if (pcmk__str_empty(OCF_RA_PATH)) {
+            crm_err("Cannot execute OCF actions because resource agent path "
```

Comment:
couldn't we simply make that a compile error?

Reply:
It might be worthwhile to validate it in configure.ac. I didn't touch it here because I thought there might be some weird use case where someone actually wants to disable OCF support, but that's probably not worth worrying about. I'll make a note to take care of that later.